### ISSUE TITLE
Auto add changeset comments when using NextView license

### ISSUE THREAD

**Issue by mikelmaron** [hotosm/osm-tasking-manager2#541]
should always have #NextView and #MapGive hashtags.

this may be covered elsewhere under the project template idea

**Comment by pyrog** [comment]
Should be in changeset source?
See #532 and #530 

#hashtag: do you mean Twitter hashtags ?

**Comment by mikelmaron** [comment]
Right, in changeset source, automatically. It's not done consistently when entirely manual.
